lib/Models/UserKeys.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+  var BaseModel, UserKeys,
+    bind = function(fn, me){ return function(){ return fn.apply(me, arguments); }; },
+    extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
+    hasProp = {}.hasOwnProperty;
+
+  BaseModel = require('../BaseModel');
+
+  UserKeys = (function(superClass) {
+    extend(UserKeys, superClass);
+
+    function UserKeys() {
+      this.all = bind(this.all, this);
+      return UserKeys.__super__.constructor.apply(this, arguments);
+    }
+
+    UserKeys.prototype.all = function(userId, fn) {
+      if (fn == null) {
+        fn = null;
+      }
+      return this.get("users/" + (parseInt(userId)) + "/keys", (function(_this) {
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'_this' is defined but never used.